include/tvm/script/ir_builder/tir/ir.h

```diff
@@ -56,7 +56,7 @@ Buffer BufferDecl(Array<PrimExpr> shape, DataType dtype, String buffer_name, Opt
  * \brief The primitive function statement.
  * \return The PrimFuncFrame.
  */
-PrimFuncFrame PrimFunc();
+PrimFuncFrame PrimFunc(const Bool& is_private);
```

Comment:
let's use `bool` here given we don't need nullability:

```suggestion
PrimFuncFrame PrimFunc(bool is_private);
```

Reply:
Thanks, I've done it